--- a/chrome/browser/ui/tabs/tab_strip_prefs.cc
+++ b/chrome/browser/ui/tabs/tab_strip_prefs.cc
@@ -34,3 +34,3 @@ void RegisterProfilePrefs(user_prefs::Pr
   registry->RegisterIntegerPref(prefs::kHeliumLayout,
-                                std::to_underlying(HeliumLayoutType::kClassic));
+                                std::to_underlying(HeliumLayoutType::kVertical));
   registry->RegisterBooleanPref(prefs::kHeliumVerticalRightAligned, false);
